server/journeys/screen/screen/view.njk

```diff
@@ -0,0 +1,40 @@
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
+{% set pageTitle = "Screen CSIP referrall" %}
+{% extends "partials/layout.njk" %}
+
+{% block innerContent %}
+  <h1 class="govuk-heading-l">Screen a CSIP referral</h1>
+
+  <form method="POST">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
+    {{ govukRadios({
+        name: "saferCustodyOutcome",
+        fieldset: {
+            legend: {
+                text: "What's the outcome of Safer Custody screening?",
+                classes: "govuk-fieldset__legend--m"
+            }
+        },
+        items: saferCustodyOutcomeItems,
+        errorMessage: validationErrors | findError('saferCustodyOutcome')
+    }) }}
+    
+    {{ govukCharacterCount({
+        label: {
+          html: '<h2 class="govuk-heading-m">Describe the reasons for this decision</h2>'
```

Comment:
This should be
```
text: "Describe the reasons for this decision",
classes: "govuk-label--m"
```
Rather than using the html itself. That way the html will update if the component does e.g. extra accessibility